recorder/tests/functionality/test_syscall.py

```diff
@@ -0,0 +1,52 @@
+import os
+import select
+
+def syscall_sum_copies(syscall):
```

Comment:
Style Choice- try having your function names be in the  imperative- e.g. `get_sum_of_memory_copies_lengths`